lib/local/print_info.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#include <unistd.h>
+
+#include "build_info.h"
+#include "git_info.h"
+
+/*
+ * Print the configuration/build info generated at build time by the executable.
+ * We do this from C and not from Fortran because Fortran has trouble with long
+ * lines.
```

Comment:
Source/example? Fortran string handling has always been incredibly ugly though...

Reply:
No specific source, just personal experience with Fortran string handling back and forth with C/C++.

Reply:
Well, I don't have any real objections given the amount of mixed code we're already doing and string formatting in fortran is horrible. (I do slightly regret picking Fortran for HANDE...)

I saw some weirdness with buffer flushing but I think that's resolved now.